mediacapture-streams/enumerating-local-media-devices/mediadevices/api-present.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+<head>
+<title>getUserMedia: test that getUserMedia is present (with or without vendor prefix)</title>
```

Comment:
Ths test checks that enumerateDevices is present, while the title, file name and so on claim to check for getUserMedia. Can you make it consistent?